### DISCUSSION TITLE
Features

### DISCUSSION THREAD

**Discussion by FBukov** [cagnulein/qdomyos-zwift#1661] [Ideas]
Hi,
Thanks for the work you are doing! I would suggest adding support for Tacx. Also, when the email with the ride summary is sent from my Echelon Sport, I am getting elevation gain when there was none, it is just a normal "free ride". Also, a Max Watt Output for 20 min could be included, not just max and average. Also Watt per kg Max Average for 20 min.
Thanks!!!

**Comment by cagnulein** [comment]
elevation gain is only when you follow a gpx file for example

i will check the other points
thanks

Il giorno ven 22 set 2023 alle 19:46 FBukov ***@***.***> ha
scritto:

**Reply by FBukov** [reply]
I simply turn on the bike and the app and start peddaling. I did not follow any pre-made workout, it still shows elevation gain. I am on IOS

**Comment by cagnulein** [comment]
yes i mean elevation will be calculated only with gpx files or zwift,
that's why is 0, it's by design

Il giorno ven 22 set 2023 alle 20:11 FBukov ***@***.***> ha
scritto:

**Reply by FBukov** [reply]
I get that, when I use Zwift, it will show elevation from there. But when I used just QZwift, without connecting to any other app, elevation used to be 0. Now it shows elevation gain every time

**Reply by FBukov** [reply]
and, it is very linear, form zero to 645 in a very steady line, like there is an constant incline of 1 or 2 %

**Comment by cagnulein** [comment]
so i guess i'm evaluating elevation from resistance. do you have a debug log about this?

**Comment by cagnulein** [comment]
yes probably as the resistance level? anyway send me a debug log so i will
check it

Il giorno ven 22 set 2023 alle 20:20 FBukov ***@***.***> ha
scritto:

**Reply by FBukov** [reply]
how do I send a debug log? :)

**Comment by cagnulein** [comment]
https://github.com/cagnulein/qdomyos-zwift/wiki/How-do-i-get-the-debug-log-in-case-something-doesn't-work%3F


Il giorno ven 22 set 2023 alle 21:41 FBukov ***@***.***> ha
scritto:
